feathr-impl/src/main/scala/com/linkedin/feathr/offline/swa/SlidingWindowAggregationJoiner.scala

```diff
@@ -137,6 +140,10 @@ private[offline] class SlidingWindowAggregationJoiner(
     // If the skip_missing_features flag is set, we will skip joining the features whose is not mature, and maintain this list.
     val notJoinedFeatures = new mutable.HashSet[String]()
 
+    // If the add default col for missing data flag is turned off,
```

Comment:
Nit: could we edit the comment to have add_default_col_for_missing_data as one word. I got confused on reading the first time.